### ISSUE TITLE
dp-probe: add subcommand to show ingress stats for UL and DL.

### ISSUE THREAD

**Issue by pshelar** [magma/magma#7252]

**Comment by pshelar** [comment]
Steps to get the stats in DL direction:
1. find the IMSI ip address.
2. find related flows where nw_dst  is the ip-address and its not arp.
```
root@magma:~# ovs-ofctl dump-flows gtp_br0 table=0
 cookie=0x0, duration=940.091s, table=0, n_packets=5239, n_bytes=2572872, priority=10,ip,in_port=LOCAL,nw_dst=192.168.128.12 actions=load:0xc400003f->NXM_NX_TUN_ID[],load:0xa0002d0->NXM_NX_TUN_IPV4_DST[],load:0x3->NXM_NX_REG8[],load:0x1->NXM_NX_REG9[],mod_dl_dst:ff:ff:ff:ff:ff:ff,load:0xbc5b39eed80e9->OXM_OF_METADATA[],resubmit(,1)
 cookie=0x0, duration=940.092s, table=0, n_packets=364, n_bytes=35672, priority=10,ip,in_port=mtr0,nw_dst=192.168.128.12 actions=load:0xc400003f->NXM_NX_TUN_ID[],load:0xa0002d0->NXM_NX_TUN_IPV4_DST[],load:0x3->NXM_NX_REG8[],load:0x1->NXM_NX_REG9[],mod_dl_dst:ff:ff:ff:ff:ff:ff,load:0xbc5b39eed80e9->OXM_OF_METADATA[],resubmit(,1)
```
3. extract the stats for LOCAL and mtr0 port:
```
LOCAL: n_packets=5239, n_bytes=2572872
mtr0: n_packets=364, n_bytes=35672
```

**Comment by pshelar** [comment]
steps for UL:
1. find the IP for imsi
2. find the ingress tunne-id from this flow. it is already done in dp-probe in function get_egress_tunid_and_port()
```
 cookie=0x0, duration=940.091s, table=0, n_packets=5239, n_bytes=2572872, priority=10,ip,in_port=LOCAL,nw_dst=192.168.128.12 actions=load:0xc400003f->NXM_NX_TUN_ID[],load:0xa0002d0->NXM_NX_TUN_IPV4_DST[],load:0x3->NXM_NX_REG8[],load:0x1->NXM_NX_REG9[],mod_dl_dst:ff:ff:ff:ff:ff:ff,load:0xbc5b39eed80e9->OXM_OF_METADATA[],resubmit(,1)
```
3. from this search flow that has the tun_id field:
```
 cookie=0x0, duration=940.091s, table=0, n_packets=7404, n_bytes=4603779, priority=10,tun_id=0x1,in_port="g_d002000a" actions=mod_dl_src:02:00:00:00:00:01,mod_dl_dst:ff:ff:ff:ff:ff:ff,load:0x1->NXM_NX_REG9[],load:0xbc5b39eed80e9->OXM_OF_METADATA[],resubmit(,1)
```
4. print the stats:
```
UL stats: n_packets=7404, n_bytes=4603779
```

**Comment by pshelar** [comment]
merged to 1.6
